app/src/main/java/fr/free/nrw/commons/upload/UploadItem.java

```diff
@@ -23,6 +23,7 @@
     private final String createdTimestampSource;
     private final BehaviorSubject<Integer> imageQuality;
     private boolean hasInvalidLocation;
+    private final Uri contentUri;
```

Comment:
Would you mind copy/pasting the getContentUri() javadoc to here?

Reply:
done.